src/main/java/uk/gov/justice/hmpps/probationteams/health/HealthInfo.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.hmpps.probationteams.health
+
+import org.springframework.boot.actuate.health.Health
+import org.springframework.boot.actuate.health.HealthIndicator
+import org.springframework.boot.info.BuildProperties
+import org.springframework.stereotype.Component
+
+/**
+ * Adds version data to the /health endpoint. This is called by the UI to display API details
+ */
+@Component
+class HealthInfo(buildProperties: BuildProperties) : HealthIndicator {
+  private val version: String = buildProperties.version
+
+  override fun health(): Health = Health.up().withDetail("version", version).build()
+}
```

Comment:
We've got a bug where we aren't logging version number on start up.

Moving to make this more consistent with the version in the kotlin template in the hope the bug might fall out